docs/installation/server-config/index.rst

```diff
@@ -3,6 +3,12 @@
 Configuring KSQL Server
 -----------------------
 
+.. toctree::
```

Comment:
What's confusing here is that it makes you believe that only these three subsections exist on this page, but in fact there are a couple of others as well (e.g. "Available KSQL config settings" section that covers auto.offset.reset).

In other words:
- The internal TOC listed here is confusing because it is incomplete.
- Relatedly, the target internal TOC/structure for this page isn't quite right yet? This page has sub-sections "Setting Default Properties", "Setting Per-Session Properties", "KSQL Configuration Parameters" (and then we have the security, CCloud, and Avro/SchemaRegistry ones as well), which IIRC isn't our target structure.

Could we clarify this please?

Reply:
i want to keep this high-level TOC PR focused on only TOC, and have proposed a fix to this section in this existing PR: https://github.com/confluentinc/ksql/commit/29ba8d94caa8f9ec22b75f380bca76bb6c48c39a